conf/fsb-types.yaml

```diff
@@ -1,12 +1,39 @@
 - key: SAC
+  schemeId: GovernmentStatisticalService
```

Comment:
@vglushak How far into your story are you? Do you think you can PR that change separately so it's integrated into this branch?